Align event images for the Community Calls

spier opened the pull request:
I noticed that the images for the Community Calls on the [Events page](https://innersourcecommons.org/events/) did not have the same size.
This lead to the other content below the image (title etc) to be misaligned. See screenshot.

![misaligned-event-images](https://user-images.githubusercontent.com/163029/139911211-ce7a6c04-03eb-48b2-a525-7e49effbedb0.jpg)

This PR fixes that issue for the existing events. 

While at it, I also made various other changes in the same area of the code:

- Re-create all all event images for the Community Calls (CCs) to 960x540
- Reduce the file size of all image, to make the page load faster. (we are loading 8 images just for the CCs, so the images got to be reasonably small)
- Rename images to form `meetup-2021-05-04.jpg`
- Consolide filename extension to `.jpg` (came in various froms before)
- Rename community events for file format `meetup-YYYY-mm-dd.md`, to bring them into natural order.
- Remove date info from the beginning of the event description. When the event has already taken place, the date information isn't that helpful for readers anymore. Also the date info is already available in the image of the title slide.
- Removing utm campaign info from one eventbrite link (we don't have that info for all other CCs)
- Remove `featured: false` as that is the default

rrrutledge commented:
Ready for review now, Tom?

tsadler1988 commented:
Good spot @rrrutledge!

rrrutledge commented:
Superseded by https://github.com/InnerSourceCommons/innersourcecommons.org/pull/206

lenucksi commented:
> Superseded by #206

Shouldn't it be closed then?

spier committed:
Re-create all CC title slides in the exact same size

spier committed:
Consolidating image filenames to a single format

spier committed:
Remove date info from the beginning of the event description.
After the event the date information isn't that helpful for readers.
Also the date info is already available in the screenshot.

spier committed:
Renaming images to form YYYY-mm-dd.

spier committed:
Rename community events for file format meetup-YYYY-mm-dd.md, to bring them into natural order.

spier committed:
Removing utm campaign info from eventbrite link (we don't have that info for all other CCs)

spier commented:
Here how the resulting events page looks like.
![event-images-fixed](https://user-images.githubusercontent.com/163029/139913410-89033055-93e4-475e-b26b-4a6f8ea89462.jpg)

claredillon reviewed (approved):
Thanks so much for this Sebastian!

spier commented:
Thanks for review and merge!

I think this is a great improvement already.

There are still some other UI glitches in this area that could still be improved:

1. Bring the images of the Summits to the same size, or find other ways to make sure that event title etc are aligned
2. How to align the text if the title breaks into 2 rows?
3. How to align the Open/Watch buttons horizontally?  Maybe that would get fixed together with (2) automatically?
4. How to prevent repetitive intros like "Join us for an InnerSource Commons Community Call ..."
